File: 2st/Yeonwoo/잘라서 배열로 저장하기.py
def solution(my_str, n):
    answer = []
    mo = len(my_str)//n
    na = len(my_str)%n
    if len(my_str)%n==0:
        for i in range(1,mo+1):
            answer.append(my_str[(i-1)*n:i*n])
    else:
        for i in range(1,mo+2):
            answer.append(my_str[(i-1)*n:i*n])
    return answer

File: 4st/Hani/programmer/잘라서 배열로 저장하기.java
class Solution {
    public String[] solution(String my_str, int n) {
        String[] answer = {};
        if (my_str.length() % n != 0) {
            answer = new String[my_str.length() / n + 1];
        } 
        else {
            answer = new String[my_str.length() / n];
        }
        
        int j = 0;
        for (int i = 0; i < answer.length; i++) {
            
            if (my_str.substring(j, my_str.length() - 1).length() < n) {
                answer[i] = my_str.substring(j, my_str.length());
            }
            else {
                answer[i] = my_str.substring(j, j + n);
            }
            j += n;
        }
        return answer;
    }
}